packages/loader/container-loader/src/test/serializedStateManager.spec.ts

```diff
@@ -20,90 +21,105 @@ import {
 	IVersion,
 	MessageType,
 } from "@fluidframework/protocol-definitions";
-import { ITelemetryLoggerExt, createChildLogger } from "@fluidframework/telemetry-utils";
+import { MockLogger } from "@fluidframework/telemetry-utils";
 import { type IPendingContainerState, SerializedStateManager } from "../serializedStateManager.js";
 
 type ISerializedStateManagerDocumentStorageService = Pick<
 	IDocumentStorageService,
 	"getSnapshot" | "getSnapshotTree" | "getVersions" | "readBlob"
 >;
 
-interface IPendingMessage {
-	type: "message";
-	referenceSequenceNumber: number;
-	content: string;
-}
+const failProxy = <T extends object>() => {
```

Comment:
rather than copy, please move this a new file in the test folder, and reuse from there